### ISSUE TITLE
User not updatable

### ISSUE THREAD

**Issue by Syzco** [Capstone-US-Army-PEOSTRI/peostri-lms#70]
``` <-- PUT /api/v1/admin/users/list/EgQLWgHgQzmBrCYeckk4AA
key dScnaeu4RojyvNZJT-P2AA is a KEY or ID but DNE
expected key of {"type":"array","instance":"fkey","foreignApi":"[object Object]","default":"","getIdKeepAsRef":"true","name":"users"}, got dScnaeu4RojyvNZJT-P2AA, trying bfs
ERROR: PUT:/api/v1/admin/users/list/EgQLWgHgQzmBrCYeckk4AA User.parseGet 400:
  Message: Invalid key value
  Verbose: [dScnaeu4RojyvNZJT-P2AA] is not a valid string entry
APIError400: Invalid key value
    at User.IErrorable.error (C:\Users\brade\Documents\Coursework\COP4935\peostri-lms\backend\build\lms\errors.js:63:56)   
    at User.parseGet (C:\Users\brade\Documents\Coursework\COP4935\peostri-lms\backend\build\api\v1\DataManager.js:344:28)  
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 2)
    at async Team.mapEachField (C:\Users\brade\Documents\Coursework\COP4935\peostri-lms\backend\build\api\v1\DataManager.js:127:40)
    at async Team.verifyAddedDocument (C:\Users\brade\Documents\Coursework\COP4935\peostri-lms\backend\build\api\v1\DataManager.js:255:15)
    at async Team.parseGet (C:\Users\brade\Documents\Coursework\COP4935\peostri-lms\backend\build\api\v1\DataManager.js:403:13)
    at async Promise.all (index 0)
    at async User.mapEachField (C:\Users\brade\Documents\Coursework\COP4935\peostri-lms\backend\build\api\v1\DataManager.js:127:40)
  --> PUT /api/v1/admin/users/list/EgQLWgHgQzmBrCYeckk4AA 400 322ms 29b```

**Comment by henry-512** [comment]
the key [dScnaeu4RojyvNZJT-P2AA] should be a team id. there is no team with this id.

**Comment by Syzco** [comment]
User is updatable.

@henry-512 Can you change the admin/user/:id route only give me an array of Team ID's as opposed to resolving them?

Once that is done, you can close this issue.

**Comment by henry-512** [comment]
should be fixed with #67
